jupyter-extension/src/plugins/mount/mountDrive.ts

```diff
@@ -2,16 +2,22 @@ import {ModelDB} from '@jupyterlab/observables';
 import {Contents, ServerConnection} from '@jupyterlab/services';
 import {PartialJSONObject} from '@lumino/coreutils';
 import {Signal, ISignal} from '@lumino/signaling';
+import {showErrorMessage} from '@jupyterlab/apputils';
 import {DocumentRegistry} from '@jupyterlab/docregistry';
 import {URLExt} from '@jupyterlab/coreutils';
 import {requestAPI} from '../../handler';
-import {Paging} from './paging';
 import {MOUNT_BROWSER_PREFIX} from './mount';
 
-// How many files to render in the FileBrowser UI per page
-const MAX_NUM_CONTENTS_PAGE = 100;
-// How many items to request per page
-const PAGINATION_NUMBER = 400;
+// How many Content li are visible at a given time for the user to scroll.
+const VISIBLE_CONTENT_LIS = 500;
+// How many visible Content li that are padded on the top and bottom of the core set of visible Content li.
+// This padding enables the user to scroll while the Contents index changes.
+const VISIBLE_CONTENT_LI_PADDING = 200;
+// How many items to request per page.
+const PAGINATION_NUMBER = 1000;
+// How tall in pixels a Content li is.
+const LI_HEIGHT_PX = 25;
```

Comment:
Is this safe to set as a constant? This seems like something we don't really have control over -- could we extract this info from the browser at runtime?

Reply:
Good point! I can grab the clientHeight from the first contentsNode.childNodes[0] instead of hard coding this.